writing-guidelines.md

```diff
@@ -0,0 +1,146 @@
+
+When writing down your thoughts, it helps to be critical throughout, but there's no need to get it perfect right away. You can write down a few sentences, reread them, and see if it makes sense. Is it accurate? Is it complete? Is this the most concise and clear way this could be written? Does it fit in the structure I have laid out? Switching to a different task and revisiting your writing later on can also help bring clarity.
+
+The most important thing is to not be too content too quickly, and keep iterating until you feel like you're not changing much anymore and that you have achieved the kind of quality that you would like your name to be associated with. 
+
+## 6. Don't forget
+
+### 6.1. Grammar
+
+Even though it has nothing to do with the content, having poor grammar is one of the easiest ways to make your article look bad. In general, the reasoning is that if you can't get something as simple as that right, what else is wrong with the article?
+
+Agree with that reasoning or not, improving your grammar has become much easier nowadays thanks to the help of various tools. Many text editors already have this functionality built-in, and there are dedicated tools such as [grammarly](https://www.grammarly.com/) that offer such services for free.
+
+If you're asking someone to review your article, make sure to grammar check your writing every time before submitting it as a draft to avoid wasting their time on easy-to-correct mistakes.
+
+### 6.2. Plagiarism
+
+> Plagiarism is presenting someone else’s work or ideas as your own, with or without their consent, by incorporating it into your work without full acknowledgement.
+> 
+> https://www.ox.ac.uk/students/academic/guidance/skills/plagiarism
+
+When presenting an idea that is not yours, you'll have to choose between using your own wording or quoting/paraphrasing the original author. You should always strive to rephrase a topic in your own words (e.g. by phrasing from it an alternative perspective, or in simpler terms, or by combining it with other topics). Sometimes rephrasing is not possible or sensible. In that case, it is okay to quote or paraphrase, but you *always* need to clearly indicate which content is quoted/paraphrased and reference the source.
+
+Even though plagiarism is often not illegal, it is almost guaranteed to blemish your reputation if and when it is discovered. Especially in an open-source community such as Bitcoin's, be generous with your references. It ensures the original author receives credit for their work where it is due. As an additional upside, ample linking to high-quality sources is a great way to make your own article more useful. Even if your writing is sufficiently original, it's usually a good idea to consistently reference the sources that inspired your thinking.
```

Comment:
Maybe add that is shows a laziness because it takes credit for other's work without creating value.

Reply:
Fixed in 50ed939, thanks!